src/main/kotlin/com/charlatano/scripts/BombTimer.kt

```diff
@@ -92,13 +93,14 @@ private data class BombState(var hasBomb: Boolean = false,
         if (gettingDefused) {
 //            sb.append("GettingDefused : $gettingDefused \n")
             sb.append("CanDefuse : $canDefuse \n")
+            // Redundant as the UI already shows this, but may have a use case I'm missing
             sb.append("TimeToDefuse : ${formatFloat(timeLeftToDefuse)} ")
         }
         return sb.toString()
     }
 
 
     private fun formatFloat(f: Float): String {
-        return "%.2f".format(f)
+        return "%.3f".format(f)
```

Comment:
We are running this 8 ms intervals, so we already lost 8/10 of 3rd decimal. So kind a pointless.